x-pack/plugins/security_solution/public/overview/pages/overview.test.tsx

```diff
@@ -298,7 +298,8 @@ describe('Overview', () => {
         expect(wrapper.find('[data-test-subj="empty-page-endpoint-action"]').exists()).toBe(false);
       });
 
-      it('shows Endpoint get ready button when ingest is enabled', () => {
+      /** Skip endpoint for now as the option has been removed via https://github.com/elastic/kibana/issues/107682 */
+      it.skip('shows Endpoint get ready button when ingest is enabled', () => {
```

Comment:
Are we planning on adding it back? If not, just remove the test I would say.

Reply:
I removed the test, I agree that we should just take it out.